### DISCUSSION TITLE
Migrate to symfony/console from laminas\console

### DISCUSSION THREAD

**Discussion by eastern-ravindra** [symfony/symfony#50485] [Q&A]
Previously, we were using `laminas/console` package for console applications, but that package is already abandoned.
So, we are now using `symfony/console` package.
One scenario was working with `laminas/console` package was, I can use the command
```
commandName (requiredArgument1|requiredArgument2|requiredArgument3|requiredArgument4):argumentReference [--option1=] --toption2Id= --type=(type1|type2|type3)
```
If we provide wrong argumentReference(i.e. not from requiredArgument1, requiredArgument2, requiredArgument3, requiredArgument4) in `laminas/console` package, it will automatically validate for the same.

Is there any in built functionality for this type of validation in `symfony/console` package?

**Comment by ghost** [comment]
What is an argumentReference? Please describe this concept because you are asking Symfony community about a feature from another framework.
If you ask whether symfony/console validates that an argument exists then the answer is yes.

**Reply by eastern-ravindra** [reply]
Actually, `argumentReference` is any name for the argument.
How can we validates that argument exists in symfony/console? I didn't find anything about this in the documentation.

**Reply by ghost** [reply]
When I run console command with nonexistent argument I get an error. You don't?
